nservicebus/scalability-and-ha/distributor/index.md

```diff
@@ -210,4 +212,4 @@ Check out [John Breakwell's blog](http://blogs.msdn.com/b/johnbreakwell/archive/
 
 If the Distributor goes down, even if its worker nodes remain running, they do not receive any messages. It is important to run the Distributor on a cluster that has its its queues configured as clustered resources.
 
-Since the Distributor does not do CPU or memory intensive work, you can often put several Distributor processes on the same clustered server. Be aware that the network IO may end up being the bottleneck for the Distributor, so take into account message sizes and throughput when sizing your infrastructure.
\ No newline at end of file
+Since the Distributor does not do CPU or memory intensive work, you can often put several Distributor processes on the same clustered server. Be aware that the network IO may end up being the bottleneck for the Distributor, so take into account message sizes and throughput when sizing your infrastructure.
```

Comment:
> when sizing your infrastructure

I know you didn't write it now, but wouldn't it be better to say "when configuring" or "when fine-tuning" your infrastructure?